### PR TITLE
Add investment likelihood of landing and priority

### PR DESCRIPTION
**Pull request by reupen**

### CODE REVIEW

File: datahub/investment/test/test_views.py [hunk 1]
```diff
@@ -804,6 +809,48 @@ def test_patch_project_conditional_failure(self):
             'fdi_type': ['This field is required.']
         }
 
+    def test_patch_likelihood_of_landing_too_low(self):
+        """Test updating a project a likelihood_of_landing above 100."""
```
Comment (elcct) [inline datahub/investment/test/test_views.py]: I think comment should be like:
```
"""Test updating a project a likelihood_of_landing equal or above 0."""
```

File: datahub/investment/models.py [hunk 2]
```diff
@@ -14,6 +14,12 @@
 
 MAX_LENGTH = settings.CHAR_FIELD_MAX_LENGTH
 
+PRIORITIES = Choices(
```
Comment (canni) [inline datahub/investment/models.py]: SHouldn't that be placed on a Model class?

File: datahub/investment/test/test_views.py (hunk 1 above)
Comment (reupen) [inline datahub/investment/test/test_views.py]: Fixed :+1:

File: datahub/investment/models.py (hunk 2 above)
Comment (reupen) [inline datahub/investment/models.py]: OK, I've moved them.

File: datahub/investment/test/test_views.py (hunk 1 above)
Comment (elcct) [inline datahub/investment/test/test_views.py]: That should be `below 0` actually :) sorry

**Pull request merged**